.github/workflows/openxla_pin_update_weekly.yml

```diff
@@ -0,0 +1,79 @@
+name: Weekly Code Update PR
+on:
+  workflow_dispatch: # Manual trigger
+  schedule:
+    - cron: '0 9 * * 1'  # Each Monday 9:00AM (PDT time)
```

Comment:
minor suggestion: comment out the cron schedule until you have a successful manual run or two.

Reply:
Thanks, updated